### PR TITLE
Vertical slice fixes

### PR DESCRIPTION
**Pull request by mac2648**

### CODE REVIEW

File: Source/Redemption/Private/AudioManager.cpp [hunk 1]
```diff
@@ -65,8 +65,9 @@ void AAudioManager::StartRandomAmbientSounds()
 // Implementation to start playing ambient music
 void AAudioManager::PlayAmbientMusic()
 {
-    if (AmbientMusicCue != nullptr && !bIsInCombat)
+    if (AmbientMusicCue != nullptr && bIsInCombat)
     {
+        bIsInCombat = false;
```
Comment (Entwistle-905) [inline Source/Redemption/Private/AudioManager.cpp:70]: Can you explain me why did you change this?

**Review by Entwistle-905 — APPROVED** [review]

File: Source/Redemption/Private/AudioManager.cpp (hunk 1 above)
Comment (mac2648) [reply Source/Redemption/Private/AudioManager.cpp:70]: the bug before was that when we create the timer to go back to safe music we change the In combat to be false, and if we entered the vision of a rat again it would be false and because of that it would restart the music. To solve the bug I changed where it changes the bool variable

Comment (Entwistle-905) [reply Source/Redemption/Private/AudioManager.cpp:70]: Understood. Thank you for the explanation.

File: Source/Redemption/Private/PauseMenuWidget.cpp [hunk 2]
```diff
@@ -26,10 +26,11 @@ void UPauseMenuWidget::Exit()
 	//Opens the "MainMenu" Map
 	UGameplayStatics::OpenLevel(GetWorld(), "MainMenu");
 
+	/*
 	APlayerController* Controller = UGameplayStatics::GetPlayerController(GetWorld(), 0);
 	Controller->SetInputMode(FInputModeUIOnly());
 	Controller->bShowMouseCursor = false;
-
+	*/
```
Comment (Fedic) [inline Source/Redemption/Private/PauseMenuWidget.cpp:33]: Can this code be removed now?

Comment (mac2648) [reply Source/Redemption/Private/PauseMenuWidget.cpp:33]: yes, I will create a branch later to delete all commented code and take out UE_LOGs

**Review by deepakpaul77 — APPROVED** [review]

**Review by Fedic — APPROVED** [review]

**Review by thakkarnandish — APPROVED** [review]
